actions/kustomize/action.yml

```diff
@@ -49,6 +49,10 @@ runs:
 
           echo "Downloading kustomize ${VERSION} for ${OS}/${ARCH}"
           KUSTOMIZE_TARGET_FILE="kustomize_${VERSION}_${OS}_${ARCH}.tar.gz"
+          if [[ "$OS" == "windows" ]]; then
```

Comment:
For backwards compatibility, this should likely also introduce a version check :grimacing: